pdr_backend/lake/test/resources.py

```diff
@@ -54,18 +63,44 @@ def _filter_gql_tables_config(record_config: Dict, record_filter: str) -> Dict:
 
 
 @enforce_types
-def _lake_ss(parquet_dir, feeds, st_timestr=None, fin_timestr=None):
+def _lake_ss(lake_dir, feeds, st_timestr=None, fin_timestr=None):
     return LakeSS(
         {
             "feeds": feeds,
-            "parquet_dir": parquet_dir,
+            "lake_dir": lake_dir,
             "st_timestr": st_timestr or "2023-06-18",
             "fin_timestr": fin_timestr or "2023-06-21",
             "timeframe": "5m",
         }
     )
 
 
+@enforce_types
+def get_filtered_timestamps_df(
+    df: pl.DataFrame, st_timestr: str, fin_timestr: str
+) -> pl.DataFrame:
+    return df.filter(
+        (pl.col("timestamp") >= UnixTimeMs.from_timestr(st_timestr))
+        & (pl.col("timestamp") <= UnixTimeMs.from_timestr(fin_timestr))
+    )
+
+
+def _clean_up_persistent_data_store(tmpdir, table_name=None):
+    # Clean up PDS
+    persistent_data_store = PersistentDataStore(str(tmpdir))
+
+    # Select tables from duckdb
+    table_names = persistent_data_store.get_table_names()
+
+    # Drop the tables
+    if table_name in table_names:
```

Comment:
easier:
table_names = [table_name] if table_name else persistent_data_store.get_table_names()
for table in table_names:
             persistent_data_store.duckdb_conn.execute(f"DROP TABLE {table}")